### PR TITLE
Hca phone component

### PR DESCRIPTION
**Pull request by akainic**
Create a component for Phone. To be used anywhere that a phone number is asked for. Also moved grid classes out of `ErrorableTextInput` because some text inputs need to be different sizes than other inputs. Those classes should go in the question component itself.

### CODE REVIEW

File: _health-care/_js/components/HealthCareApp.jsx [hunk 1]
```diff
@@ -61,7 +61,10 @@ class HealthCareApp extends React.Component {
             isWhite: false
           },
 
-          veteranAddress: {}
+          veteranAddress: {
+            homePhone: '555-555-5555',
+            mobilePhone: '555-555-5555'
```
Comment (awong-dev) [inline _health-care/_js/components/HealthCareApp.jsx]: Just for my own OCDness, use a different number here. 1111 maybe? That way we can tell quickly if we've cross wires between the fields.

File: _health-care/_js/components/form-elements/ErrorableTextInput.jsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ class ErrorableTextInput extends React.Component {
     }
 
     return (
-      <div className={`usa-input-grid usa-input-grid-medium ${this.props.errorMessage ? 'usa-input-error' : ''}`}>
+      <div className={`${this.props.errorMessage ? 'usa-input-error' : ''}`}>
```
Comment (awong-dev) [inline _health-care/_js/components/form-elements/ErrorableTextInput.jsx]: In this case, you don't need a template string anymore, and you can change '' to undefined so no class attribute is even generated.

File: _health-care/_js/components/personal-information/Phone.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import ErrorableTextInput from '../form-elements/ErrorableTextInput';
+import { isValidPhone } from '../../utils/validations.js';
+
+/**
+ * Input component for collecting a Phone number.
+ *
+ * Validates the input data. Does NOT consider "invalid" phone such as
```
Comment (awong-dev) [inline _health-care/_js/components/personal-information/Phone.jsx]: phone such -> phone numbers such

File: _health-care/_js/components/personal-information/Phone.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    );
+  }
+}
+
+Phone.propTypes = {
+  label: React.PropTypes.string,
+  value: React.PropTypes.string.isRequired,
+  onValueChange: React.PropTypes.func.isRequired,
+};
+
```
Comment (awong-dev) [inline _health-care/_js/components/personal-information/Phone.jsx]: spurious new line.

File: _health-care/_js/utils/validations.js [hunk 5]
```diff
@@ -12,7 +12,12 @@ function isValidDate(day, month, year) {
     date.getFullYear() === Number(year);
 }
 
+function isValidPhone(value) {
+  return /^\d{3}-\d{3}-\d{4}$/.test(value);
```
Comment (awong-dev) [inline _health-care/_js/utils/validations.js]: TODO this for now, but I wonder if we should just use a library like
  https://www.npmjs.com/package/validator
or
  https://www.npmjs.com/package/iz

both seem to be relatively well used. Especially the first...(36,287 downloads in the last day!)

**Comment by awong-dev** [comment]
Nice!  Sent you a couple of small comments.

**Comment by akainic** [comment]
@awong-dev Made fixes, PTAL!

**Comment by awong-dev** [comment]
LGTM!

**Pull request merged**